optaweb-employee-rostering-standalone/pom.xml

```diff
@@ -147,7 +147,7 @@
                   <goal>start</goal>
                 </goals>
                 <configuration>
-                  <name>Run application</name>
+                  <name>ERP backend</name>
```

Comment:
I think that hardly anybody recognizes what "ERP" is; moreover, it's not just the backend, but the entire standalone application bundled in a single JAR file.

Reply:
True